### DISCUSSION TITLE
Improve display of icons in Toonz Raster levels

### DISCUSSION THREAD

**Discussion by beeheemooth** [opentoonz/opentoonz#6361] [Ideas]
The display of the contents of the frame in the Level Strip of the vector level is limited to a frame that covers all the drawn details in this level frame. Thanks to this, the user can easily find the level by the icon.

In Toonz Raster Level, the entire work surface is displayed in the Level Strip icon. If at the level drawings are small details, then in icons they are almost invisible on small monitors and it becomes difficult to distinguish levels by icons (I turn on / off the column display to make sure that I have selected the right one).

  Is it possible to make the icons display not the entire working surface of the level, but only the area from savebox?

![ezgif-2-8b5cbb92cad1](https://user-images.githubusercontent.com/25724133/80859997-7fb34380-8c6d-11ea-8f6f-08c6d1036531.gif)

**Comment by DarrenTAnims** [comment]
This would be really helpful.

**Comment by artisteacher** [comment]
That would be fantastic.

**Comment by gab3d** [comment]
the idea is good. I'm not sure how easy would it be to calculate the drawn area, though...

another good feature regarding readability of Level Strip thumbnails would be to be able to hover on one of them, pressing Ctrl or some other key, to make OT show a (floating) bigger version of the thumbnail to one side of it. 
That could also help in identifying between very similar drawings with small differences.

**Comment by ave4** [comment]
It would be very nice if opentoonz would create a raster file the size of the drawn area. And not 1080x1920, which is why even with a small figure drawn a very large raster file is obtained.

**Comment by DarrenTAnims** [comment]
> It would be very nice if opentoonz would create a raster file the size of the drawn area. And not 1080x1920, which is why even with a small figure drawn a very large raster file is obtained.

The default size starts off at the size of your camera. But then if you want a different size, you can change it in your preferences, in the Drawing section. And if you realise that the drawing level size it still too large, you can always shrink it manually. OpenToonz doesn't have dynamic raster drawing areas and probably never will.
